SPECS/kata-containers/kata-containers.spec

```diff
@@ -76,11 +62,15 @@ BuildRequires:  kernel
 BuildRequires:  busybox
 BuildRequires:  cargo
 BuildRequires:  rust
+BuildRequires:  device-mapper-devel
+BuildRequires:  clang
 
 Requires:       busybox
 Requires:       kernel
 Requires:       libseccomp
-Requires:       qemu-virtiofsd
+# Must match the version specified by the `assets.virtiofsd.version` field in
+# %{SOURCE0}/versions.yaml.
+Requires:       virtiofsd = 1.8.0
 
 Conflicts:      kata-agent
 Conflicts:      kata-ksm-throttler
```

Comment:
Description and Summary. May want to update a bit at least in terms of versioning

Reply:
To fix